src/lib/components/streams/streams-manager.svelte

```diff
@@ -113,7 +117,19 @@
     })
 
     async function onSearch(): Promise<void> {
-        await searchAllChannels(query)
+        await searchAllChannels(query, { limit: DEFAULT_SDK_CLIENT_REQUEST_LIMIT })
```

Comment:
You are adding a limit here but not the identity manager, you want to add it also there perhaps?